code/code/misc/spell_info.cc

```diff
@@ -1517,7 +1517,7 @@ void buildSpellArray() {
       LEARN_DO_2, START_DO_NO, LEARN_DO_NO, LEARN_DIFF_SKILLS, 0.0, 0, 0);
 
   discArray[SKILL_BERSERK] =
-    new spellInfo(SKILL_WARRIOR, DISC_WARRIOR, DISC_SOLDIERING, STAT_EXT,
+    new spellInfo(SKILL_WARRIOR, DISC_WARRIOR, DISC_BRAWLING, STAT_EXT,
```

Comment:
Since adv berserking is in brawling I moved this. There should be no impact due to this though